internal/bucket/access.go

```diff
@@ -25,3 +25,25 @@ func (b *bucketImpl) GetBucketItems() []*bt.BucketItem {
 func (b *bucketImpl) ContentExists(id string) bool {
 	return b.contentCache[id] == nil
 }
+
+func (b *bucketImpl) GetContent() []*BucketContent {
+	var content []*BucketContent = make([]*BucketContent, len(b.contentCache))
+	for _, v := range b.contentCache {
+		if v.ContentType == bt.ResourceIdentifier_CID {
+			content = append(content, v)
```

Comment:
using `append` with a slice that has a preconfigured size will cause the slice to be longer than necessary. So in this `content` will have size `len(b.contentCache) * 2`

This applies below as well